tests/AnalyserTest.php

```diff
@@ -526,6 +526,30 @@ public function testDevPathInsideProdPath(): void
         self::assertEquals($this->createAnalysisResult(2, []), $result);
     }
 
+    public function testPharSupport(): void
+    {
+        require_once __DIR__ . '/data/not-autoloaded/phar/org/package/inner.phar';
```

Comment:
You should not add binary phar like that to repository, because it's completely unreadable. It's much better to create the phar in tests on the fly.